ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-path-selection.j2

```diff
@@ -10,6 +10,9 @@ router path-selection
 {%     if router_path_selection.peer_dynamic_source is arista.avd.defined %}
    peer dynamic source {{ router_path_selection.peer_dynamic_source }}
 {%     endif %}
+{%     if router_path_selection.tcp_mss_ceiling.ipv4_segment_size is arista.avd.defined %}
+   tcp mss ceiling {{ router_path_selection.tcp_mss_ceiling.ipv4_segment_size }} {{ router_path_selection.tcp_mss_ceiling.direction | arista.avd.default("ingress") }}
```

Comment:
```suggestion
   tcp mss ceiling ipv4 {{ router_path_selection.tcp_mss_ceiling.ipv4_segment_size }} {{ router_path_selection.tcp_mss_ceiling.direction | arista.avd.default("ingress") }}
```

Also, this is valid:

```
ceos1(config-dynamic-path-selection)#show active
router path-selection
   tcp mss ceiling ipv4 ingress
```

Reply:
fixed by adding "auto" option for ipv4_segment_size